src/ServicePulse.Host/vue/src/components/PageHeader.vue

```diff
@@ -22,7 +22,7 @@ const displayDanger = computed(() => {
   <nav class="navbar navbar-expand-lg navbar-inverse navbar-dark">
     <div class="container-fluid">
       <div class="navbar-header">
-        <a class="navbar-brand" href="/">
+        <a class="navbar-brand" :href="baseUrl + routeLinks.dashboard">
```

Comment:
We want the responsibility of creating links to be in `routeLinks.ts`, if we need to fully qualify a url, you can use the `useLink` component.

Reply:
`routeLinks.ts` is where all internal links are defined.
However, because we are still in a mix of Angular and Vue, we also define Angular links in there for easier of maintenance.